### PR TITLE
Add hot deploy paragraph to README

### PR DESCRIPTION
**Pull request by jhadvig**
@bparees @mnagy I've updated the README as we spoke. PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,3 +180,19 @@ To set environment variables, you can place them as a key value pair into a `.st
 file inside your source code repository.
 
 Example: DATABASE_USER=sampleUser
+
+Hot deploy
+---------------------
+Hot deploy is not supported in this image. To support it, your application needs to be built with one of modules that reloads the server every time changes in source code are done inside the running container. Those modules are:
```
Comment (bparees) [inline README.md]: built with?  more like "Run by" right?

Comment (jhadvig) [inline README.md]: @bparees well you need to install the `supervisor` or `nodemon` in order to be able to run it.

Comment (bparees) [inline README.md]: yeah but it feels a little weird to say it's "built with supervisor", you're just installing supervisor (via npm install) as a separate action from building your app, in my mind.

**Comment by bparees** [comment]
[test]

**Comment by jhadvig** [comment]
[test]

**Comment by mfojtik** [comment]
LGTM

**Pull request merged**